utils/makefile.python

```diff
@@ -99,7 +99,8 @@ quiet_cmd_pyinstall = INSTALL   $2
 quiet_cmd_pyenvinstall = PYENV     install $2
       cmd_pyenvinstall = \
 	if ! cat $(PY_ENV)/requirements.sha256 2>/dev/null | sha256sum --check --status 2>/dev/null; then \
-		$(PY_ENV_BIN)/python -m pip $(PIP_VERBOSE) install -e $2$(PY_SETUP_EXTRAS) ;\
+		$(PY_ENV_BIN)/python -m pip $(PIP_VERBOSE) install $$(cat ./requirements.txt | grep -i pyyaml); \
+		$(PY_ENV_BIN)/python -m pip $(PIP_VERBOSE) install -e $2$(PY_SETUP_EXTRAS) &&\
 		sha256sum requirements*.txt > $(PY_ENV)/requirements.sha256 ;\
```

Comment:
OK, you only want to create a `$(PY_ENV)/requirements.sha256 ` when pip succeeds .. make sense to me.

Does it make sense to have a:

    rm -f $(PY_ENV)/requirements.sha256 ;

before pip installs?

Reply:
Done in fe3085d68e965732ca7291a14225b34a7e046e92